charts/vela-core/crds/core.oam.dev_applications.yaml

```diff
@@ -1491,9 +1499,17 @@ spec:
                     type: object
                   finished:
                     type: boolean
+                  lastExecuteTime:
+                    format: date-time
+                    type: string
+                  message:
+                    type: string
                   mode:
                     description: WorkflowMode describes the mode of workflow
                     type: string
+                  nextExecuteTime:
+                    format: date-time
+                    type: string
                   startTime:
                     format: date-time
                     type: string
```

Comment:
* What will happen if the CRD is not updated?
* Will it cause the infinite loop？

Reply:
If this field is empty, it'll be reconciled after a second.

Reply:
You should store the time in Configmap